crates/swc_css_parser/src/parser/input.rs

```diff
@@ -6,7 +6,7 @@ use swc_css_ast::{Token, TokenAndSpan, Tokens};
 use super::PResult;
 use crate::error::{Error, ErrorKind};
 
-pub trait ParserInput {
+pub trait ParserInput: Clone + Iterator<Item = TokenAndSpan> {
     type State: Debug;
 
     fn next(&mut self) -> PResult<TokenAndSpan>;
```

Comment:
I think we can safely remove this method

Reply:
@kdy yes, I write it above, I just want to remove it in other PR + do buffer refactor for perf reasons, so I postpone it